services/streammanager/kafka/kafkamanager.go

```diff
@@ -67,13 +68,16 @@ func init() {
 	loadConfig()
 	loadCertificate()
 	pkgLogger = logger.NewLogger().Child("streammanager").Child("kafka")
+	// sarama.Logger = log.New(os.Stdout, "[Sarama] ", log.LstdFlags)
 }
 
 func loadConfig() {
 	clientCertFile = config.GetEnv("KAFKA_SSL_CERTIFICATE_FILE_PATH", "")
 	clientKeyFile = config.GetEnv("KAFKA_SSL_KEY_FILE_PATH", "")
+	// kafkaBatchingEnabled = config.GetBool("Router.kafka.enableBatching", false)
 	config.RegisterDurationConfigVariable(time.Duration(10), &kafkaDialTimeout, false, time.Second, []string{"Router.kafkaDialTimeout", "Router.kafkaDialTimeoutInSec"}...)
 	config.RegisterDurationConfigVariable(time.Duration(2), &kafkaWriteTimeout, false, time.Second, []string{"Router.kafkaWriteTimeout", "Router.kafkaWriteTimeoutInSec"}...)
+	config.RegisterBoolConfigVariable(false, &kafkaBatchingEnabled, false, []string{"Router.kafka.enableBatching"}...)
```

Comment:
If it is only a single variable you can pass it as is rather than a string array

Reply:
yes